src/components/Card.tsx

```diff
@@ -12,47 +12,30 @@ type Item = {
   description: string;
 };
 
-type CardProps = {
-  item: Item;
-  key: number;
-};
-
-const Card: FC<CardProps> = ({ item, key }) => {
```

Comment:
This looks like an expensive `noop` to me. We're binding multiple event listeners, one for every instance of `Card`, and it doesn't seem to be used for anything.